### PR TITLE
docs: add `formkit` under Beta Releases section #284

### PR DESCRIPTION
**Pull request by sun-mota**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #284 

## Summary:

Adding Formkit's install and release page until Beta Releases section.

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

## Summary by Sourcery

Add Formkit component to the project, including installation and release pages, and update the documentation to reflect these additions under the Beta Releases section.

New Features:
- Introduce Formkit component with installation and release pages under the Beta Releases section.

Documentation:
- Add documentation for Formkit installation and release information.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -847,6 +851,12 @@ function App() {
               <Route path="/svelteSetup" element={<SvelteDemoDocs />} />
 
 
+              {/* Auro Formkit */}
+              <Route path="/components/auro/formkit" element={<AuroFormkitInstall />} />
+              <Route path="/components/auro/formkit/install" element={<AuroFormkitInstall />} />
+              <Route path="/components/auro/formkit/releases" element={<AUroFormkitRelease />} />
```
Comment (jordanjones243) [inline src/App.js:857]: Typo here

File: src/App.js [hunk 2]
```diff
@@ -847,6 +851,12 @@ function App() {
               <Route path="/svelteSetup" element={<SvelteDemoDocs />} />
 
 
+              {/* Auro Formkit */}
+              <Route path="/components/auro/formkit" element={<AuroFormkitInstall />} />
```
Comment (jordanjones243) [inline src/App.js:855]: Is there a reason why we have duplicate pages showing the install instructions?

File: src/components/side-nav.js [hunk 3]
```diff
@@ -101,6 +101,12 @@ export default function SideNav(props) {
         { linkTitle: "The <slot> element", route: "/support/slots", parent: true },
         { linkTitle: "Accessibility statement", route: "/a11y-statement", parent: true },
       ]
+    },
+    {
+      header: "Beta Releases",
```
Comment (jordanjones243) [inline src/components/side-nav.js:106]: Not sure if you got specific instructions, but I would recommend moving this up near the top of the sidenav. I barely knew it was there and I think the "Beta Releases" section will be really important to consumers in the future.

**Review by jordanjones243 — CHANGES_REQUESTED** [review]

File: src/components/side-nav.js (hunk 3 above)
Comment (sun-mota) [reply src/components/side-nav.js:106]: I was instructed by Jason to put it on the bottom.

File: src/App.js (hunk 1 above)
Comment (sun-mota) [reply src/App.js]: fixed

**Review by jordanjones243 — APPROVED** [review]

**Pull request merged**